tests/infra/remote.py

```diff
@@ -11,6 +11,8 @@
 import ctypes
```

Comment:
Some changes here so that we can copy directories to/from `LocalRemote` and `SSHRemote`.